packages/picker/src/Picker.ts

```diff
@@ -406,6 +411,16 @@ export class PickerBase extends SizedMixin(Focusable) {
                           ></sp-icon-alert>
                       `
                     : nothing}
+                ${this.loading
+                    ? html`
+                          <sp-progress-circle
+                              label="Loading picker"
+                              indeterminate
+                              class="sp-picker-loading sp-picker-loading-${this
```

Comment:
This class could just be `loading`, or similar, as we're scoped to this shadow root.
```suggestion
                              class="loading sp-picker-loading-${this
```
The sizing can also be interpreted from the `:host` as I commented in the CSS, simplifying the render here.